fern/definition/medical/document.yml

```diff
@@ -177,16 +214,79 @@ types:
       - html
       - pdf
 
-  ListDocumentsResponse:
+  MedplumDocumentReference:
     properties:
-      documents: list<DocumentReference>
+      documents: list<<map<string, unknown>>
```

Comment:
this is temporary until I know how to import the actual fern fhir reference spec